### DISCUSSION TITLE
POLL: Traditional 42x42 vs Hypermodular 14x14 Rail Grid

### DISCUSSION THREAD

**Discussion by Bocian-1** [Opinionated-Blueprints/10-Books-Full-of-Rails#83] [Polls]
Currently, there are two options for a basic set of blueprints for the main railway: the standard 42x42 grid railway and the newer 14x14 hypermodular option. I'm interested in the usage of the two, especially since grids smaller than some of the pieces aren't something you see often so I'd expect some instinctual aversion to the unknown so to speak, even though having played around with it myself, I'd pick the hypermodular rail over regular 10 times out of 10. Another thing is that even with 14x14 being a 1/3 of 42x42, making some things compatible with both means especially those using 42x42 have a worse experience since everything has to snap for 14x14 and in some cases it completely misses the mark (cityblock exits align with 14x14 but only every 3rd exit aligns with 42x42). I'm not sure if anything will come out of this poll, but I might outright remove or at least split the traditional 42x42 prints into a separate book.

**Comment by brandon-gradient** [comment]
I prefer 42x42 for aesthetics, but the 14x14 are indeed quite handy. I just really dislike the diagonal spacing on 14x. I think that's because I used the old book for 1.1 for so long and really got used to how things were and if this 2.x book only ever had hypermodular I don't think this would even be a question, hypermodular is so much nicer to work with, at the cost of having to think a bit more and things looking worse.
I think it's like choosing between accepting a bit of belt spaghetti for efficiency's sake vs pure main bus.

**Reply by selectL-L** [reply]
I agree fully with this, It's also easier to actually route the rails through my own spaghetti when they're not 42x42!

**Comment by Bocian-1** [comment]
Every time I come back here it's either equal or swung in the opposite direction from the previous haha~

**Reply by brandon-gradient** [reply]
yeah currently 17 for 42x42 and 14 for 14x14!

**Comment by mkaito** [comment]
I feel like the whole point of using grid snapping is that things actually align at the other end. So 42 it is for me.

**Reply by Bocian-1** [reply]
They still align with 14x14 though? And 14x14 is more precise so you can go between cliffs and stuff...

**Reply by brandon-gradient** [reply]
Oh yeah with space age, and cliff explosives locked behind vulcanus, this does add a nice point to 14x14. Indeed there have been quite a few areas that I was only able to make feasible with 14x14. Of course I could have done some weird not good looking pathing from the exit to the station and back but I'm so in love with the new inline stations! I make an excuse to use them wherever possible

**Reply by Fevix** [reply]
> They still align with 14x14 though? And 14x14 is more precise so you can go between cliffs and stuff...

I don't think it's that big of an ask to manually route around, or over, the cliffs.

**Comment by bwoods89** [comment]
I mix them depending on my needs. I like to do bigger stretches with 42x42 like connecting new resource patches. I use the 14x14 most often when I need to route around lava or connect up a tiny scrap island on fulgora
